src/torchjd/autojac/_transform/base.py

```diff
@@ -50,12 +50,16 @@ def __call__(self, input: _B) -> _C:
     @property
     @abstractmethod
     def required_keys(self) -> set[Tensor]:
-        raise NotImplementedError
+        """
+        Returns the set of keys that the transform expects to be present in its input TensorDicts.
+        """
 
     @property
     @abstractmethod
     def output_keys(self) -> set[Tensor]:
-        raise NotImplementedError
+        """
+        Returns the set of keys that will be present in the TensorDicts returned by the transform.
```

Comment:
`TensorDicts returned by the transform` -> `output of the transform`.
If not, then maybe we want ``` `TensorDict`s```